client/jetpack-onboarding/controller.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+
+/**
+ * Internal Dependencies
+ */
+import JetpackOnboardingMain from './main';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import { setSection } from 'state/ui/actions';
+
+const removeSidebar = context => {
+	ReactDom.unmountComponentAtNode( document.getElementById( 'secondary' ) );
+	context.store.dispatch( setSection( null, { hasSidebar: false } ) );
+};
+
+export const onboarding = context => {
+	removeSidebar( context );
+
+	renderWithReduxStore(
+		React.createElement( JetpackOnboardingMain, {
+			stepName: context.params.stepName,
+		} ),
```

Comment:
Let's use JSX here -- it's sort of Calypso convention by now to prefer that over `createElement`.

Reply:
Done in 54dc9a46291bc4e805428106545715ff7bb960f5